scripts/index.js

```diff
@@ -104,4 +104,16 @@ nTimes(2, hello)
 
 function hello(){
     console.log('Hello World!')
-}
\ No newline at end of file
+}
+
+function strMultiply(str, times) {
+    let newArray = []
+    let newString = ''
+    for (i = 0; i < times; i++) {
+        newArray.push(str);
+    }
+    newString = newArray.toString();
+    return newString.replace( /,/g,"")
```

Comment:
Fancy! For an alternate technique, google for JavaScript's `.join` array method.